### PR TITLE
V. Petrat - Added Google search image test

### PR DESCRIPTION
**Pull request by VladimirPetrat**

### CODE REVIEW

File: src/main/java/com/softserveinc/ita/vpetrat/pageobjects/GoogleHomePage.java [hunk 1]
```diff
@@ -12,11 +12,11 @@ public GoogleHomePage openHomePage() {
         return this;
     }
 
-    public GoogleSearchResultPage searchFor(String request) {
+    public GoogleSearchAllResultsPage searchFor(String request) {
```
Comment (toleksyn) [inline src/main/java/com/softserveinc/ita/vpetrat/pageobjects/GoogleHomePage.java]: rename

File: src/main/java/com/softserveinc/ita/vpetrat/pageobjects/GoogleSearchAllResultsPage.java [hunk 2]
```diff
@@ -29,6 +28,15 @@
     }
 
     public String getSearchResultLinkTextForIndex(int index) {
-        return getListOfSearchResultLinks().get(index);
+        return getListOfSearchResultLinksText().get(index);
+    }
+
+    public GoogleSearchImageResultsPage showSearchResultImages() {
```
Comment (toleksyn) [inline src/main/java/com/softserveinc/ita/vpetrat/pageobjects/GoogleSearchAllResultsPage.java]: rename

File: src/main/java/com/softserveinc/ita/vpetrat/pageobjects/GoogleSearchImageResultsPage.java [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+                .click();
+        return new GoogleHomePage();
+    }
+
+    public List<String> getListOfSearchResultImagesText() {
+        return TestRunner
+                .getDriver()
+                .findElements(By.xpath("//div[@id='islmp']" +
+                        "//div[@jsaction='IE7JUb:e5gl8b;dtRDof:s370ud;R3mad:ZCNXMe;v03O1c:cJhY7b;']" +
+                        "//a[@class='VFACy kGQAp sMi44c lNHeqe WGvvNb']"))
```
Comment (toleksyn) [inline src/main/java/com/softserveinc/ita/vpetrat/pageobjects/GoogleSearchImageResultsPage.java]: locator is overcomplicated, simplify

File: src/test/java/com/softserveinc/ita/pageobjects_task/vpetrat/GoogleSearchTest.java [hunk 4]
```diff
@@ -33,4 +34,15 @@ public void testGoogleSearchAllResultLink() {
...
     }
+
+    @Test
+    public void testGoogleSearchResultImagesText() {
+        GoogleSearchImageResultsPage imageResultsPage = googleHomePage.searchFor("funny kitten").showSearchResultImages();
+        List<String> resultImagesText = imageResultsPage
+                .getListOfSearchResultImagesText();
+        imageResultsPage.returnToHomePage();
+        Assert.assertTrue(resultImagesText.size() >= 10);
+        Assert.assertTrue(resultImagesText.get(0).contains("funny") && resultImagesText.get(4).contains("funny"));
```
Comment (toleksyn) [inline src/test/java/com/softserveinc/ita/pageobjects_task/vpetrat/GoogleSearchTest.java]: split

File: src/main/java/com/softserveinc/ita/vpetrat/pageobjects/GoogleSearchImageResultsPage.java (hunk 3 above)
Comment (toleksyn) [inline src/main/java/com/softserveinc/ita/vpetrat/pageobjects/GoogleSearchImageResultsPage.java]: locator is overcomplicated, simplify

File: src/test/java/com/softserveinc/ita/pageobjects_task/vpetrat/GoogleSearchTest.java (hunk 4 above)
Comment (toleksyn) [inline src/test/java/com/softserveinc/ita/pageobjects_task/vpetrat/GoogleSearchTest.java]: split

**Review by toleksyn — CHANGES_REQUESTED** [review]

File: src/test/java/com/softserveinc/ita/pageobjects_task/vpetrat/GoogleSearchTest.java (hunk 4 above)
Comment (VladimirPetrat) [reply src/test/java/com/softserveinc/ita/pageobjects_task/vpetrat/GoogleSearchTest.java]: Split into 3 test methods?

**Pull request closed without merging**